backend/aci/server/tests/routes/apps/test_apps_get.py

```
@@ -16,6 +16,7 @@ def test_get_app(
     dummy_api_key_1: str,
     dummy_app_github: App,
 ) -> None:
+    assert False
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)